ci/submodule-sync.sh

```diff
@@ -63,6 +63,10 @@ if [[ "${cudf_sha}" == "${cudf_prev_sha}" ]]; then
   exit 0
 fi
 
+echo "Try update cudf submodule to ${cudf_sha}..."
+git add .
+git diff-index --quiet HEAD || git commit -s -m "Update submodule cudf to ${cudf_sha}"
```

Comment:
It looks like this is going to commit twice, since we're committing here and also on line 101 below.  Do we need to commit twice?  Should the second commit be via `--amend`?

Reply:
Good idea. I forgot the amend way to reduce the twice commits. Let me try.

Reply:
Since we remove the first commit, it doesn't require the --amend way because it's a single commit.